### PR TITLE
Edited Documentation for LSA

### PR DESCRIPTION
**Pull request by ChinmayMundane**
This pull request solves the following issue:
1) Removed the Acknowledgement Tag from Readme.
2) Changed Images from old LSA to current version.
3) Changed the LSA content from the table

### CODE REVIEW

File: 2_LSA/README.md [hunk 1]
```diff
@@ -17,18 +17,17 @@
 - The Light Sensor Array(LSA) is used to detect the white/black lined surface and it sends this data to the microcontroller ESP32. Thus, the purpose of line following is achieved.
 
 - Physical PCB
-<img src="assets/lsa_pcb.jpeg" width="800" height="450">
+<img src="assets/LSA_physical.jpeg" width="800" height="450">
```
Comment (ChinmayLonkar) [inline 2_LSA/README.md:20]: Add a better-quality image in png format.

File: 2_LSA/README.md [hunk 2]
```diff
@@ -17,18 +17,17 @@
...
 
 - PCB Model
-<img src="assets/lsa_pcb_model.jpeg" width="800" height="400">
+<img src="assets/LSA.png" width="800" height="400">
```
Comment (ChinmayLonkar) [inline 2_LSA/README.md:23]: use the screenshot of the updated LSA PCB in KiCAD

File: 2_LSA/README.md [hunk 3]
```diff
@@ -17,18 +17,17 @@
...
 
 ### Components of LSA
 | Component   | Description   | Quantity |
 | ----------- | ------------- | -------- |
-| LED         | Emits light   |  4       |
-| PhotoDiode  | Absorbs light |  4       |
-| IC LM324    | Op.Amp. IC    |  1       |  
-| Resistors   | Limit Current |  8       |
+| LED         | Emits light   |  5       |
```
Comment (ChinmayLonkar) [inline 2_LSA/README.md:28]: Make it IR LED and Phototransistor.

**Review by ChinmayLonkar — CHANGES_REQUESTED** [review]
Update the `Function of Components` section as per the current LSA :))

**Comment by SuperChamp234** [comment]
Closing this PR as it is stale.

**Pull request closed without merging**